src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/ActivitiesMigrationFilter.kt

```diff
@@ -17,4 +18,10 @@ data class ActivitiesMigrationFilter(
     example = "12345",
   )
   val courseActivityId: Long? = null,
+
+  @Schema(
+    description = "The date the new activity will start",
+    example = "2025-01-31",
+  )
+  val activityStartDate: LocalDate? = null,
```

Comment:
Add the date that migrated activities will start. This will be available to users in the migration UI.